packages/hig-toast/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@hig/toast",
```

Comment:
Any input on package name or directory name?

Reply:
Works for me.

Reply:
Thoughts on removing the `hig-` prefix in the directory name, to be `packages/toast`?

I assuming that every package we add in this project will be published as either `@hig/*` or `hig-*`.

Reply:
Agreed @morrisallison.